Shapes.java

```diff
@@ -1,25 +1,52 @@
-import java.util.*;
+import java.util.Scanner;
+import java.lang.Math;
 
-public class Shapes{
+class Shapes{
   public static void main(String[] args) {
     Scanner input = new Scanner(System.in);
-    System.out.print("Enter the side length: ");
+
+    System.out.print("What is the side length for squares? ");
     double side = input.nextDouble();
+
     square(side);
-  }
 
-  /*
-   * N: square
-   * P: to calculate the area of any square
-   * I: a side measure (double)
-   * R:
-   */
-  public static void square(double s){
-    //side times side
-    double a = s * s;
+    System.out.print("What is the base for triangles? ");
+    double base = input.nextDouble();
```

Comment:
Nice variable names! They clearly represent what value they're storing and their purpose.